src/main/resources/db/changelog/db.changelog-RDM-8485.xml

```diff
@@ -0,0 +1,16 @@
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+
+<changeSet id="rdm-8485" author="suman.sreenivasan@hmcts.net">
+    <sql dbms="postgresql"
+         endDelimiter="\nGO"
+         splitStatements="true"
+         stripComments="true">
+        CREATE INDEX IF NOT EXISTS idx_case_data_deceaseddateofbirth ON public.case_data USING BTREE ((TRIM(UPPER(data#>>'{deceasedDateOfBirth}'))));
+        CREATE INDEX IF NOT EXISTS idx_case_data_createdingapsfrom_dwpstate ON public.case_data USING BTREE ((TRIM(UPPER(data#>>'{createdingapsfrom,dwpstate}'))));
+        CREATE INDEX IF NOT EXISTS idx_case_data_d8petitionerlastname ON public.case_data USING BTREE ((TRIM(UPPER(data#>>'{D8PetitionerLastName}'))));
+    </sql>
+</changeSet>
```

Comment:
idx_case_data_appeal_appellant_identity_nino is missing

Reply:
Please ask Alliu to provide above 4 index sql statements if he has. I guess, {createdingapsfrom,dwpstate} required proper syntax to pick up correctly.

Reply:
CREATE INDEX idx_case_data_D8PetitionerLastName ON public.case_data USING BTREE ((TRIM(UPPER(data#>>'{D8PetitionerLastName}'))));
CREATE INDEX idx_case_data_createdInGapsFrom_dwpState ON public.case_data USING BTREE ((trim(upper((data #>> '{createdInGapsFrom}')))),(trim(upper((data #>> '{dwpState}')))));
CREATE INDEX idx_case_data_deceasedDateOfBirth ON public.case_data USING BTREE ((TRIM(UPPER(data#>>'{deceasedDateOfBirth}'))));
CREATE INDEX idx_case_data_appeal_appellant_identity_nino ON public.case_data USING BTREE ((TRIM(UPPER(data#>>'{appeal,appellant,identity,nino}'))));

Reply:
Will added these changes.